.github/workflows/cicd.yaml

```diff
@@ -40,8 +40,8 @@ jobs:
     - name: Example inference run via Docker with default config and checkpoint
       run: >
         docker run
-        -v /var/data/cicd/CICD_github_assets/myria3d_V3.3.0/inputs/:/inputs/
```

Comment:
if the version number in this path is meant to be updated with the version number of the code, it would be better to infer it from the code (like here: https://github.com/IGNF/ign-pdal-tools/blob/dev/.github/workflows/cicd_deploy.yml)

Reply:
btw, is it still 3.4.0 after updating to 3.4.7?

Reply:
Actually it is not necessary to update CICD artefacts with every patch update. Even changing minor version does not necessary change the inference behavior (in this case it does).

So infering the full version might not be the right move here. Alternatively, we may infer the Major+Minor, and make it therefore **mandatory** to create a new dir in CICD_github_assets. This would require to extract Major+Minor numbers from the full version (eithe rin _version.py) or something like
```bash
version="3.4.17"
major_minor=$(echo "$version" | cut -d. -f1-2)
echo "$major_minor"
```

Reply:
IMO it can quickly become annoying to have to update assets everytime, especially if changes triggering a change in minor versions have no impact on inference. What do you think?

Reply:
In this case it it probably too much of a hassle. If there is a change in the inference, I suppose it will raise errors in the CI, so there is no big risk at letting the user do it manually if it is from time to time